### PR TITLE
Parametrized project logo URL in header of dashboard

### PR DESCRIPTION
**Pull request by azaricstefan**
As per issue #2289 creating this PR.

I also thought about some of the solutions mentioned [here](https://github.com/laravel/framework/issues/7671) but I am not sure if we should complicate it more.

This proposed solution is using APP_URL variable from .env.

We could also leave the url hardcoded, but I didn't liked that approach.

### CODE REVIEW

**Comment by tabacitu** [comment]
Merged! It'll be available with a ```composer update``` later today, as Backpack\CRUD 4.0.24. Thank you for helping make Backpack better @azaricstefan ! 

I've just made two changes to the default value:
- I decided to use the same ```url()``` to calculate the default value, not APP_URL; the reason is that APP_URL is not properly configured by A LOT of developers, since it's only used by Laravel in one place - when creating the URL for the password reset email; I've even heard the myth that "_it's useless_"; I've found a lot of projects that don't have that set up, so I think it's safer to use ```url()```, as we did before, so we don't break people's apps; using APP_URL in the _default_ config file would have no benefit, but it would add the risk of breaking the link for some people; not worth it; of course, in _your_ config file you can use APP_URL if you prefer that;
- the default value should lead to the project root, not ```/admin```, for backwards compatibility;

In short, the change I made is very small, just to keep the default "as it were":
```diff
-    'project_logo_url' => env('APP_URL').'/admin',
+    'project_logo_url' => url(''),
```

Thanks again @azaricstefan !

**Comment by adampogany** [comment]
@tabacitu,
using url() in the config file breaks artisan commands - thus installation - for me (Laravel 6.7.0, 6.3.0):

```
In UrlGenerator.php line 120:

  Argument 2 passed to Illuminate\Routing\UrlGenerator::__construct() must be
   an instance of Illuminate\Http\Request, null given, called in /var/www/htm
  l/xxx/vendor/laravel/framework/src/Illuminate/Routing/RoutingServ
  iceProvider.php on line 63
```

**Comment by tabacitu** [comment]
Ouch! Thank you for telling us about it @adampogany . You're totally right, just pushed the fix. The config value is now a string like ```''``` or ```'admin'```, instead of ```url('')``` or ```url('admin')```. I've also renamed it to ```home_link``` - I think it's cleaner this way.

The problem was that we can't use the ```url()``` helper inside config files. I knew that when pushing the feature, even wondered _why_ it worked. I was wondering _when_ Laravel decided to load the helpers before the configs. Apparently never 😄 It was just that I didn't properly test artisan commands before pushing 4.0.24. Sorry about this!

Cheers!

**Comment by azaricstefan** [comment]
Correct, that's the reason why I didn't used url() function in config, but when you used it I thought it's working somehow for you. Better to have it then hardcoded.

home_link sounds good 👍 

@tabacitu Thanks for taking a look and merging this PR so fast!
